markdown/extensions/tables.py

```diff
@@ -85,7 +85,39 @@ def _split_row(self, row, border):
                 row = row[1:]
             if row.endswith('|'):
                 row = row[:-1]
-        return row.split('|')
+        return self._split(row, '|')
+
+    def _split(self, row, marker):
+        """ split a row of text with some code into a list of cells. """
+        if self._row_has_code(row):
+            ## fallback on old behaviour
+            return row.split(marker)
```

Comment:
According to the [coverage report](https://coveralls.io/builds/3919721/source?filename=markdown%2Fextensions%2Ftables.py), this is never tested. Might want to add a test for it.